Kickstarter-iOS/Views/Controllers/ManageViewPledgeViewController.swift

```diff
@@ -123,7 +178,18 @@ final class ManageViewPledgeViewController: UIViewController {
 
 // MARK: Styles
 
-public let viewStyle: ViewStyle = { (view: UIView) in
+private let rootScrollViewStyle = { (scrollView: UIScrollView) in
+  scrollView
+    |> \.alwaysBounceVertical .~ true
+}
+
+private let rootStackViewStyle = { (stackView: UIStackView) in
```

Comment:
In `PledgeViewController` we also have a `rootStackViewStyle`: https://github.com/kickstarter/ios-oss/blob/master/Kickstarter-iOS/Views/Controllers/PledgeViewController.swift#L402

I wonder if it be worthwhile to extract this style into our `CheckoutStyles.swift` file so they can be shared? I'd imagine that the pledge screen and the manage pledge screen should probably have consistent root stack view styles regardless.

Reply:
LOL, there's so many repeated styles..I'll try to consolidate these into something more reusable 😄